app/views/shared/_status.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="<%= dialog_column_class %>">
+  <h1>Status of your request</h1>
+  <hr />
+  <dl class='dl-horizontal dl-invert'>
+    <dt>Requested on</dt>
+    <dd><%= request.created_at.to_formatted_s(:long_ordinal) %></dd>
```

Comment:
Yeah, TBH I've seen a few date formats around the app and think we should strictly identify how we want to format dates on the particular screens.  This was a built-in rails way that at least looked something like what was in the mock.  I'll create a ticket about validating the different date formats around the app and we can throw them into i18n for referencing around the app.